backend/api/routes/jobs.py

```
@@ -523,6 +524,311 @@ async def get_instrumental_options(job_id: str) -> Dict[str, Any]:
+        return {
+            "status": "success",
+            "message": "Custom instrumental created successfully",
+            "custom_instrumental_url": storage.generate_signed_url(output_path, expiration_minutes=120),
+            "mute_regions_applied": len(result.mute_regions_applied),
+            "total_muted_duration_seconds": result.total_muted_duration_seconds,
+            "output_duration_seconds": result.output_duration_seconds,
+        }
+        
+    except Exception as e:
+        logger.error(f"Error creating custom instrumental for job {job_id}: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)